README.md

```diff
@@ -39,7 +39,7 @@ cd src/main
 ### Create user
 In order to create a cognito user with already verified password must be used the following 
 
-IMPORTANT Username MUST NOT be in email format.
+IMPORTANT Username MUST NOT be in email format. Leaving blank the email attribute will result in no action in the user registration flow, the email value MUST be set.
```

Comment:
Maybe something like this is clearer:

> The email attribute **MUST** be set, otherwise the registration flow will break and the user won't receive any email. If you don't set the email attribute at user creation time, you will have to delete and create the user again with the email attribute set

What do you think?

Reply:
I agree. Much clearer